source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -302,6 +302,20 @@ MongoClient from the one used for test operations. For example::
             replSetStepDown: 20
             force: false
 
+**Note:** The "replSetStepDown" command often fails with the following
+transient error (see `SERVER-48154`_)::
+
+  {
+    "ok" : 0,
+    "errmsg" : "Unable to acquire X lock on '{4611686018427387905: ReplicationStateTransition, 1}' within 1000ms. opId: 922, op: conn30, connId: 30.",
+    "code" : 24,
+    "codeName" : "LockTimeout",
+  }
+
+When running the "replSetStepDown" command, drivers MUST retry until the
```

Comment:
Same as above.  It's unfortunate to have to do this in a generic test runner, but I don't see the alternative.